### PR TITLE
My reservations fix

### PR DESCRIPTION
**Pull request by andking**
@Suebxl @jfnumainville @MarkSoosaipillai 
Check it out - just few corrections in Reservation Controller (Simple version) -)

### CODE REVIEW

File: app/controllers/reservations_controller.rb [hunk 1]
```diff
@@ -1,97 +1,56 @@
...
-    end
-else
-    #Display all the User's reservations for a non-admin User
-    @reservations = User.find_by(id: @user_id).reservations
- end
+   @reservations = Reservation.all
 end
 
-def show
-    if session[:curren_useer_id]
```
Comment (Suebxl) [inline app/controllers/reservations_controller.rb:29]: is the syntax correct ? shouldnt be : current_user_id

File: app/controllers/reservations_controller.rb [hunk 2]
```diff
@@ -1,97 +1,56 @@
...
+end
+
+def find_user
+    @user = current_user
 end
 
+def find_booking
+    @reservation = Reservation.find(params[:id])
 end
 end
```
Comment (Suebxl) [inline app/controllers/reservations_controller.rb:56]: is this last end not too much? delete it?

Comment (andking) [reply app/controllers/reservations_controller.rb:56]: It’s for the whole class end if I’ve counted correct

Comment (jfnumainville) [reply app/controllers/reservations_controller.rb:56]: Thanks Andrii, that looks good :-)

**Review by jfnumainville — DISMISSED** [review]

**Pull request closed without merging**